drivers/led/lp5562.c

```diff
@@ -951,6 +949,21 @@ static const struct led_driver_api lp5562_led_api = {
 	.off = lp5562_led_off,
 };
 
-DEVICE_DT_INST_DEFINE(0, &lp5562_led_init, NULL, &lp5562_led_data,
-		      &lp5562_led_config, POST_KERNEL, CONFIG_LED_INIT_PRIORITY,
-		      &lp5562_led_api);
+
+#define LP5562_INSTANTIATE(n)                                           \
+    static const struct lp5562_config lp5562_config_data_##n = {        \
+		.bus = I2C_DT_SPEC_INST_GET(n),				 \
+    };                                                                  \
+                                                                        \
+    struct lp5562_data lp5562_data_##n;                                 \
+    DEVICE_DT_INST_DEFINE(                                              \
+        n,                                                              \
+        &lp5562_led_init,                                               \
+        NULL,                                                           \
+        &lp5562_data_##n,                                               \
+        &lp5562_config_data_##n,                                        \
+        POST_KERNEL,                                                    \
+        CONFIG_LED_INIT_PRIORITY,                                       \
+        &lp5562_led_api);                                               \
```

Comment:
Please follow the style of other LED drivers:
```
#define LP5562_DEVICE(id)
...
DEVICE_DT_INST_DEFINE(id, &lp5562_led_init, NULL,
...
```

Reply:
Thanks, updated to follow the same style